osu.Game/Users/UserPanel.cs

```diff
@@ -96,6 +104,7 @@ private void load()
                 Add(background);
 
             Add(CreateLayout());
+            Add(loading = new LoadingLayer(true));
```

Comment:
I don't think there should be a loading layer for such a rare action as block, especially when it's activated from a menu. Just remove this.